sql/core/src/main/java/org/apache/spark/sql/sources/v2/writer/streaming/StreamWriter.java

```diff
@@ -39,21 +36,21 @@
    * If this method fails (by throwing an exception), this writing job is considered to have been
    * failed, and the execution engine will attempt to call {@link #abort(WriterCommitMessage[])}.
    *
-   * To support exactly-once processing, writer implementations should ensure that this method is
-   * idempotent. The execution engine may call commit() multiple times for the same epoch
-   * in some circumstances.
+   * The execution engine may call commit() multiple times for the same epoch in some circumstances.
```

Comment:
Somewhere in this file, add docs about what epochId means for MB and C execution.

Reply:
+1